packages/ecc-client-ga4gh-tes/.eslintrc

```diff
@@ -14,6 +14,7 @@
     "@typescript-eslint/no-unused-vars": ["error"],
     "import/no-unresolved": "off",
     "import/no-extraneous-dependencies": "off",
+    "@typescript-eslint/no-non-null-assertion": "off",
```

Comment:
This make the code less `type-safe`, where is this being conflicted?

Reply:
from FAST docs `this.name = (event.target! as HTMLInputElement).value;`, this kind of line was throwing the above-mentioned error, hence I thought it might be better to include the rule in, nowhere else the rule is being used only in places like these.

Reply:
- Change `"off"` to `"warn"` so that we can see if it's due to the above-mentioned code or from some "type" mistake. But that will make the console really messy.
- Instead of using  `this.name = (event.target! as HTMLInputElement).value` use `this.name = (event.target as HTMLInputElement).value` which works, but  since former was how the official docs do it I didn't use the later.

What should I do?

Reply:
Yeah, you can use the latter form in the 2nd point! I don't think `event.target` will be null in its lifecycle.

Reply:
please check and resolve :+1: